### PR TITLE
feat(bil): savings plan

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: console/billing/additional-content/understanding-savings-plans.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    - Block Storage SSD
+    - Block Storage volumes
+    - Block Storage Low Latency volumes
+
+The following resources are **not** covered by the Savings Plan discount:
+
+    - Instances H100, RENDER, L40, L4, DEV1, GP1, START1, X64, POP-WIN
+    - Instance and Block Storage snapshots 
+    - Local storage 
+
```
Comment (jpaturel-scw) [inline console/billing/additional-content/understanding-savings-plans.mdx:113]: Maybe here we can add a note indicating that other scaleway products are not included, and that the list of products included or not in the savings plans may change regularly.

File: console/billing/how-to/purchase-savings-plan.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  paragraph: This page explains how to purchase a Savings Plan via the Scaleway console.
+tags: billing payment invoice savings plans
+dates:
+  validation: 2024-06-10
+  posted: 2024-06-10
+categories:
+  - billing
+---
+
+Savings Plans provide a flexible pricing structure designed to optimize costs. By committing to consistent usage volume, you unlock substantial discounts compared to pay-as-you-go models, resulting in significant long-term savings. Savings plans are available for Compute resources.
```
Comment (jpaturel-scw) [inline console/billing/how-to/purchase-savings-plan.mdx:16]: replace usage volume by usage amount

File: console/billing/how-to/purchase-savings-plan.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+For more details about how Savings Plans work, how they are billed and which resources are covered, refer to the [Understanding Savings Plans](/console/billing/additional-content/understanding-savings-plans.mdx) documentation page.
+
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid payment method added to your account.
```
Comment (jpaturel-scw) [inline console/billing/how-to/purchase-savings-plan.mdx:25]: Maybe here we can add that the user have to validate his identity to purchase a SP

File: console/billing/how-to/purchase-savings-plan.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+4. Select a savings plan.
+    <Message type="note">
+      Currently, Scaleway only offers Savings Plans for compute resources.
+    </Message>
+5. Select the duration of the plan. You can choose between 12, 24 and 36 months.
+    <Message type="important">
+      Keep in mind that once purchased, Savings Plans cannot be canceled or exchanged.
+    </Message>
+6. Enter the monthly commitment amount. The amount must be at least €1000.
```
Comment (jpaturel-scw) [inline console/billing/how-to/purchase-savings-plan.mdx:43]: at least €100

File: console/billing/how-to/purchase-savings-plan.mdx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+7. Review the details of your Savings Plan.
```
Comment (jpaturel-scw) [inline console/billing/how-to/purchase-savings-plan.mdx:44]: And the discount that will be applied to your commitment amount. thats represente the user savings based on the duration and commitment amount

File: console/billing/additional-content/understanding-savings-plans.mdx [hunk 6]
```diff
@@ -47,7 +47,7 @@ Currently, Scaleway only offers Savings Plans for compute resources.
 
 ### Commitment amount
 
-The monthly commitment amount for a Savings Plan must be at least €1000. The amount you specify will determine in which discount bracket your plan falls.
+The monthly commitment amount for a Savings Plan must be at least €100 and at most €10 000. The amount you specify will determine in which discount bracket your plan falls.
```
Comment (jsudraud) [inline console/billing/additional-content/understanding-savings-plans.mdx:50]:
```suggestion
The monthly commitment amount for a Savings Plan must be at least €100 and at most € 10,000. The amount you specify will determine in which discount bracket your plan falls.
```

**Review by jsudraud — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**